src/logic/notifications/notificationBuilder.js

```diff
@@ -231,7 +231,7 @@ export const showSnackbar = (notification: Notification, enqueueSnackbar: Functi
   enqueueSnackbar(notification.message, {
     ...notification.options,
     // eslint-disable-next-line react/display-name
-    action: (key) => (
+    action: key => (
```

Comment:
prettier 2.0 forces brackets around function arguments. Do you mind running `prettier:fix`?

Reply:
done!